src/api/data.js

```diff
@@ -56,4 +56,155 @@ export const events = {
             image: "https://picsum.photos/200/300",
         }
     ],
-};
\ No newline at end of file
+};
+
+export const people = {

```

Comment:
Can we just call actual data here instead of use this test api data?
We have an endpoint in the back-end already implemented (with real names and class standings) for getting club lead info...

Reply:
Yep!